lib/darksky.rb

```diff
@@ -0,0 +1,33 @@
+require 'httparty'
+
+class Darksky
+  def self.darksky_info(coordinates)
```

Comment:
All of your methods are class methods. Is there a reason you chose to do that. What benefits could there be to using instances?